packages/api/src/command/medical/patient/converage-assessment-get.ts

```diff
@@ -24,17 +25,73 @@ export type CoverageAssessment = {
   mrSummaryUrl: string | undefined;
 };
 
-export async function getCoverageAssessment({
+export type PatientWithCoverageAssessment = Patient &
+  Partial<Omit<CoverageAssessment, "patientId">>;
+
+export async function getCoverageAssessments({
+  cxId,
+  patients,
+}: {
+  cxId: string;
+  patients: Patient[];
+}): Promise<PatientWithCoverageAssessment[]> {
+  const { log } = out(`getCoverageAssessments - cxId ${cxId}`);
+  const patientsWithAssessment: PatientWithCoverageAssessment[] = [];
+
+  async function getCoverageAssessmentWrapper({
+    cxId,
+    patient,
+    patientsWithAssessment,
+    log,
+  }: {
+    cxId: string;
+    patient: Patient;
+    patientsWithAssessment: PatientWithCoverageAssessment[];
+    log: typeof console.log;
+  }): Promise<void> {
+    try {
+      const coverageAssessment = await getCoverageAssessment({ cxId, patient, log });
+      patientsWithAssessment.push({
+        ...patient,
+        ...coverageAssessment,
+      });
+    } catch (error) {
+      const msg = `Failed to get coverage assessment. Cause: ${errorToString(error)}`;
+      capture.error(msg, {
+        extra: {
+          cxId,
+          patientId: patient.id,
+          context: "coverage-assessment.get",
+        },
+      });
+      patientsWithAssessment.push(patient);
+    }
+  }
+
+  await executeAsynchronously(
+    patients.map(patient => {
+      return { cxId, patient, patientsWithAssessment, log };
+    }),
+    getCoverageAssessmentWrapper,
+    { numberOfParallelExecutions: 50 }
```

Comment:
ditto: `getCoverageAssessment` might generate some non-trivial load (on the FHIR server), so maybe have a lower number of parallel invocations here?

Reply:
Done.